Dockerfile_ubuntu20.04_mpi

```diff
@@ -0,0 +1,210 @@
+#Image: mspass/mspass
+#Version: 0.0.1
+
+FROM ghcr.io/seisscoped/container-base:ubuntu20.04_mpi
```

Comment:
On the second thought, this ubuntu20.04_mpi container is redundant here, because MsPASS needs the jupyter component in most cases, and the MPI + Jupyter is exactly what the `ghcr.io/seisscoped/container-base:latest` has. I guess we could also remove the jupyter component in this one and replace it with something more lightweight (like ipython). I am not sure if that is necessary though.